File: kotlin-app/app/src/main/java/com/example/handyhub/ui/model/MasterCardUiModel.kt
package com.example.handyhub.ui.model

data class MasterCardUiModel(
    val id: Int,
    val fullName: String,
    val phone: String,
    val email: String,
    val categoryId: Int?,
    val categoryName: String,
    val description: String,
    val expYears: Int,
    val priceFrom: Double,
    val ratingAvg: Double,
    val reviewsCount: Int,
    val avatarUrl: Int,
)

File: react-native-app/src/ui/models/MasterCardUiModel.ts
export type MasterCardUiModel = {
  id: number;
  fullName: string;
  categoryName: string;
  categoryIds: number[];
  description: string;
  searchText: string;
  expYears: number;
  priceFrom: number;
  ratingAvg: number;
  reviewsCount: number;
  avatarUrl?: string;
};

export type MasterCardItem = MasterCardUiModel;
